src/redux/reducers/differentialExpression/differentialExpressionUnselectDeletedOption.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-param-reassign */
+import produce from 'immer';
+import { getCellSetKey } from 'utils/cellSets';
+
+const differentialExpressionUnselectDeletedOption = produce((draft, action) => {
+  const { key: deletedKey } = action.payload;
+
+  Object.keys(draft.comparison.group).forEach((comparisonType) => {
+    const comparisonGroup = draft.comparison.group[comparisonType];
+
+    Object.keys(comparisonGroup).forEach((comparisonKey) => {
+      if (getCellSetKey(comparisonGroup[comparisonKey]) !== deletedKey) return;
```

Comment:
Here there's the same pattern of using `Object.keys` but then getting the value instead of using `Object.entries` directly